### ISSUE TITLE
Ensemble: Allow metric support - Pickling issue 

### ISSUE THREAD

**Issue by muellerdo** [frankkramer-lab/aucmedi#144]
Tensorflow keras metrics are not pickle-able (you get cryptic errors about missing state updates).

Thus, passing metrics into subprocesses of ensemble learning functions (like composite, stacking and bagging) are not possible.

Find solution fix for this issue.

Starting point: Finding out why tensorflow keras metrics throws errors if you pickle them (with dill).
